Doc/c-api/unicode.rst

```diff
@@ -679,88 +688,6 @@ APIs:
    .. versionadded:: 3.3
 
 
-Deprecated Py_UNICODE APIs
-""""""""""""""""""""""""""
-
-.. deprecated-removed:: 3.3 3.12
```

Comment:
Should we remove the deprecated functions from 3.11 documentation?

Reply:
No. It is just a mistake of resolving merge conflict.